src/content/en/updates/2019/07/web-components-time-to-upgrade.md

```diff
@@ -0,0 +1,219 @@
+*   If you're not sure what you're using, don't worry. See 
+    [Help! I don't know what APIs I'm using!](#use-the-v0-polyfills)
+
+## Back to the future: disabling the v0 APIs {#back-to-the-future-disabling-the-v0-apis}
+
+To test your site with the v0 APIs disabled, you need to start Chrome from the command line with 
+the following flags:
+
+```bash
+--disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports
+```
+
+You'll need to exit Chrome before starting it from the command line. If you have Chrome Canary 
+installed, you can run the test in Canary while keeping this page loaded in Chrome. (Or load this 
+page in Firefox, Safari, or Edge while you're testing in Chrome.)
+ 
+To test your site with v0 APIs disabled:
+
+1.  If you're on Mac OS, browse to `chrome://version` to find the executable path for Chrome. 
+    You'll need the path in step 3.
+1.  Quit Chrome.
+1.  Restart Chrome with the command-line flags: 
+
+    `--disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports` 
+
+    For instructions on starting Chrome with flags, see 
+    [Run Chromium with flags.](https://www.chromium.org/developers/how-tos/run-chromium-with-flags) 
+    For example, on Windows, you might run:
+
+    ```bash
+    chrome.exe --disable-blink-features=ShadowDOMV0,CustomElementsV0,HTMLImports
+    ```
+
+1.  To double check that you have started the browser correctly, open a new tab, open the DevTools 
+    console, and run the following command:
+
+    ```
+    console.log(
+    "Native HTML Imports?", 'import' in document.createElement('link'),
+    "Native Custom Elements v0?", 'registerElement' in document, 
+    "Native Shadow DOM v0?", 'createShadowRoot' in document.createElement('div'));
+    ```
+
+    If everything is working as expected, you should see:
+
+    ```
+    Native HTML Imports? false Native Custom Elements v0? false Native Shadow DOM v0? false
+    ```
+
+    If the browser reports true for any or all of these features--something's wrong. Make sure 
+    you've fully quit out of Chrome before restarting with the flags.
+
+1.  Finally, load your app and run through the features. If everything works as expected, you're 
+    done. 
+
+## Use the v0 polyfills {#use-the-v0-polyfills}
+
+If your site isn't working on Chrome with the v0 APIs disabled, chances are you aren't loading the 
+polyfills. There are a couple of possibilities:
+
+*   You're not loading the polyfills at all. In this case, your site should fail on other browsers, 
+    like Firefox and Safari.
+*   You're loading the polyfills conditionally based on browser sniffing. In this case, your site 
+    should work on other browsers. Skip ahead to Load the polyfills.
+
+### Install the v0 polyfills {#install-the-v0-polyfills}
+
+The Web Components v0 polyfills were never published to the npm registry. You have a couple of 
+choices when it comes to installing the polyfills:
+
+*   If your existing project uses bower, use bower to install the v0 polyfills.
```

Comment:
Shorten to 'To install with bower'. To restore the parallelism maybe change the other to 'To install from the zip file.'

Reply:
Hm. I feel like this misses essential information, because if weren't already using bower, you'd need to install it, initialize the project, and then install... And no one's likely to do that now, with a deprecated tool like Bower, unless they're already using it. 

So, I see two ways to go here:

1.  I could move the "if you're using bower" part to the introduction, and say,

    The Web Components v0 polyfills were never published to the npm registry. If your project uses Bower, you can install them using Bower. If not, you'll need to install them directly from GitHub.

    - To install with bower ...
    - To install from a zip file ...

Of I could move the branch to the bullet list, like:

- If your project uses Bower:
    _bower install instructions_
- If your project doesn't use Bower:
    _zip install instructions_

WDYT?

Reply:
Your call. My real purpose was to 'omit needless words' as Shrunk and White put it.